run_one.js

```diff
@@ -0,0 +1,27 @@
+const Joi = require("joi");
+const pkg = require("../../package.json");
```

Comment:
When I run `npm run one_test` I get an error because it can't find `'../../package.json'`, which makes sense because they are at the same level... 

A script run with `npm run ...` runs in the root directory of the package.